src/components/shared/header/header.jsx

```diff
@@ -20,148 +21,206 @@ const icons = {
-                  <ul
-                    className=" rounded-2xl bg-white p-3.5"
-                    style={{ boxShadow: '0px 4px 10px rgba(26, 26, 26, 0.2)' }}
-                  >
-                    {items.map(({ iconName, text, description, to }, index) => {
-                      const Icon = icons[iconName];
-                      return (
-                        <li
-                          className={clsx(index !== 0 && 'mt-3.5 border-t border-t-gray-4 pt-3.5')}
-                          key={index}
-                        >
-                          <Link
-                            className="flex items-center whitespace-nowrap hover:text-primary-2"
-                            to={to}
+const Header = forwardRef(
+  (
+    {
+      theme,
+      isMobileMenuOpen,
+      onBurgerClick,
+      isSignIn,
+      isSticky,
+      withBottomBorder,
+      isDocPage,
+      onSearchClick,
+    },
+    ref
+  ) => (
+    <header
+      className={clsx(
+        'safe-paddings absolute top-0 left-0 right-0 z-40 w-full lg:relative lg:h-14',
+        theme === 'black' && 'lg:bg-black',
+        theme === 'white' && 'bg-white',
+        isSticky && 'sticky top-0 z-50 md:relative',
+        withBottomBorder && 'border-b border-gray-7'
+      )}
+      ref={ref}
+    >
+      <Container className="flex items-center justify-between py-3.5" size="md">
+        <Link className="hidden xl:block" to="/">
+          <span className="sr-only">Neon</span>
+          {theme === 'white' && (
+            <img
+              className="h-9 2xl:h-8"
+              src={logoBlack}
+              width={114}
+              height={32}
+              alt=""
+              aria-hidden
+            />
+          )}
+          {theme === 'black' && (
+            <img
+              className="h-9 2xl:h-8"
+              src={logoWhite}
+              width={114}
+              height={32}
+              alt=""
+              aria-hidden
+            />
+          )}
+        </Link>
+        <nav className="xl:absolute xl:top-1/2 xl:left-1/2 xl:-translate-x-1/2 xl:-translate-y-1/2">
+          <ul className="flex space-x-12 2xl:space-x-10 lg:hidden">
+            {MENUS.header.map(({ to, text, items }, index) => (
+              <li className={clsx(items?.length > 0 && 'group relative')} key={index}>
+                <Link
+                  className={clsx(
+                    items?.length > 0 &&
+                      'relative pr-3.5 before:absolute before:top-[7px] before:right-0 before:h-0 before:w-0 before:border-4 before:border-[transparent] before:transition-colors before:duration-200 group-hover:text-primary-2 group-hover:before:border-t-primary-2',
+                    items?.length > 0 && theme === 'white' && 'before:border-t-black',
+                    items?.length > 0 && theme === 'black' && 'before:border-t-white'
+                  )}
+                  to={to}
+                  theme={theme === 'white' ? 'black' : 'white'}
```

Comment:
If `theme === 'white'` then `'black'`?

Reply:
One is the background theme, another is Link color text theme.

Reply:
🤯